crates/delta/src/util/test.rs

```diff
@@ -101,6 +101,9 @@ impl TestHarness {
 
         let mut stream = self.sub.on_message();
         while let Some(item) = stream.next().await {
+            // Not sure what should be done if we get a RedisError
```

Comment:
This is fine, if Redis is having issues then there are fundamental issues with the environment that need to be resolved.